### DISCUSSION TITLE
What is this FieldInput file doing here?

### DISCUSSION THREAD

**Discussion by jbruni** [scandipwa/scandipwa#4388] [General]
Why not delete the src/component/FieldInput folder entirely, since it is not used anymore?

https://github.com/scandipwa/scandipwa/blob/3127544ea6f02e5e9c9613448d8c8997c6aff516/packages/scandipwa/src/component/FieldInput/FieldInput.container.js#L15

**Comment by jbruni** [comment]
Unrelated, but let me report anyway.

Typo here:

https://github.com/scandipwa/scandipwa/blob/master/packages/scandipwa/src/type/Category.type.js#L57

**Comment by zans-laksa** [comment]
Thanks for noticing. Fixed.
